### PR TITLE
Add recipe for `jaxsim`

### PR DESCRIPTION
**Pull request by flferretti**
This is a `grayskull`-generated recipe for `jaxsim` pure-Python package.

PyPI: https://pypi.org/project/jaxsim/
Repo: https://github.com/ami-iit/jaxsim

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by flferretti** [comment]
@diegoferigo can you please confirm you want to be added as a maintainer?

**Comment by diegoferigo** [comment]
> @diegoferigo can you please confirm you want to be added as a maintainer?

Yes, thanks for opening the PR!

**Comment by flferretti** [comment]
@traversaro can you please confirm you want to be added as a maintainer?

**Comment by traversaro** [comment]
> @traversaro can you please confirm you want to be added as a maintainer?

Yes!

**Comment by flferretti** [comment]
@conda-forge/help-python @conda-forge/staged-recipes PR ready for review, thanks! 
The Windows failure is expected as JAX is not available on Windows, moreover this is a noarch recipe, so it should not be a problem. The OSX failure seems related to external reasons

File: recipes/jaxsim/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{% set name = "jaxsim" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ami-iit/jaxsim/archive/v{{ version }}.tar.gz
```
Comment (xhochy) [inline recipes/jaxsim/meta.yaml:9]: Please build from the sdist from PyPI. Then you wouldn't need the `SETUPTOOLS_SCM_PRETEND_VERSION` below.

Comment (flferretti) [reply recipes/jaxsim/meta.yaml]: Thanks @xhochy, solved in 12f17c3a7fe9e2ce2d1895de3e7eef84b659c1b2

**Review by xhochy — APPROVED** [review]

**Pull request merged**